lib/moment-range.js

```diff
@@ -353,6 +353,27 @@ moment.range = function(start, end) {
 moment.range.constructor = DateRange;
 
 /**
+ * Build a date range between a specified interval and now (or a specified date).
+ *
+ * @param {String} interval The type of interval
+ * @param {Number} [count=1] The number of intervals
+ * @param {(Moment|Date} [date=moment()] The date to use
+ *
+ * @return {!DateRange}
+ */
+moment.range.fromInterval = function(interval, count = 1, date = moment()) {
+  if (!moment.isMoment(date)) date = moment(date);
```

Comment:
Yeah, I mean I was trying to be consistent with his library (which is hard!), and he never calls that.  But it _is_ a great idea.  So I'll do it and he can learn from it!